deploy/crontab.yml

```diff
@@ -77,5 +77,5 @@
 
   - cron:
       name: "Update materialized view"
-      minute: "*/15"
+      minute: "*/1"
```

Comment:
Confirmed this takes about 5 seconds, so we're ok to merge this whenver. Suggest we wait until #2389 is merged